src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -27,6 +26,7 @@ private static void setupPresetTests(Application application) {
 				DriverFeature.getDriverManager());
 
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
+		application.addTest("Tern test",selectTestFigureOptionListener);
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.